CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v6.0.0](https://github.com/voxpupuli/puppet-extlib/tree/v6.0.0) (2022-01-14)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-extlib/compare/v5.3.0...v6.0.0)
+
+**Breaking changes:**
+
+- Drop EoL Ubuntu 16.04 [\#197](https://github.com/voxpupuli/puppet-extlib/pull/197) ([bastelfreak](https://github.com/bastelfreak))
+- Drop EoL Debian 8/9 [\#196](https://github.com/voxpupuli/puppet-extlib/pull/196) ([bastelfreak](https://github.com/bastelfreak))
+- Drop EoL RHEL/CentOS 6 support [\#195](https://github.com/voxpupuli/puppet-extlib/pull/195) ([bastelfreak](https://github.com/bastelfreak))
+- modulesync 5.1.0 [\#193](https://github.com/voxpupuli/puppet-extlib/pull/193) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
This is a breaking change because it contairs `Drop EoL Puppet 5 support`.

Would be cool to see this in the changelog?

Reply:
hmm they are tagged backwards-incompatible  i would have thought the rake task should handle that but not to familiar with that.

Reply:
wait on a second look theses are listed in the breaking changes section, so im not sure what the issue is?

Reply:
Just a matter of updating the PR #193 title to say that it removes Puppet 5 support and re-generate the changelog.

Reply:
done